tests/analog/test_pyq_vs_pulser.py

```diff
@@ -64,16 +65,61 @@ def test_analog_op_run(n_qubits: int, spacing: float, op: AbstractBlock) -> None
 
     circuit = QuantumCircuit(register, block)
 
-    model_pyqtorch = QuantumModel(circuit, backend=BackendName.PYQTORCH)
+    wf_pyq = run(circuit, values=values, state=init_state, backend=BackendName.PYQTORCH)
+    wf_pulser = run(circuit, values=values, state=init_state, backend=BackendName.PULSER)
 
-    model_pulser = QuantumModel(
-        circuit,
-        backend=BackendName.PULSER,
-        diff_mode=DiffMode.GPSR,
+    assert equivalent_state(wf_pyq, wf_pulser, atol=ATOL_DICT[BackendName.PULSER])
+
+
+def get_random_rot(param: str, qubit_support: tuple[int]) -> AbstractBlock:
+    return AnalogRot(
+        duration=param,
+        omega=torch.rand(1),
+        delta=torch.rand(1),
+        phase=torch.rand(1),
+        qubit_support=qubit_support,
     )
 
-    wf_pyq = model_pyqtorch.run(values=values, state=init_state)
-    wf_pulser = model_pulser.run(values=values, state=init_state)
+
+@pytest.mark.parametrize("spacing", [8.0, 15.0])
+@pytest.mark.parametrize(
+    "block",
+    [
+        # Currently fails because pyq does the rotations in parallel
+        # and Pulser does sequentially, to be fixed.
+        # kron(
+        #     get_random_rot("x", qubit_support=(0,)),
```

Comment:
can we make them xfail so we dont forget about them?

Reply:
Well I'm literally going to fix it right away

Reply:
ok lgtm